appendix-basic65.tex

```diff
@@ -1264,9 +1267,12 @@ \subsection{CHAR}
                 (neither printed nor interpreted).
 
                 Notice that the start position of the string has
-                different units in the horizontl and vertical directions
+                different units in the horizontal and vertical directions
                 horizontal is in [columns] and vertical is in [pixels].
 
+                Refer to the {\bf CHR\$} command on page \pageref{chrcommand}
+                for more information.
```

Comment:
I felt a little confused initially at why I was being directed to CHR$ for more information. Looking the command over, ok, I guess I understand why, to learn more details of various possible CHR$() values that could be injected into the string used by the CHAR command. So I'm ok to leave it in, just wanted to share my initial reaction :)